server/controllers/chartsController.js

```diff
@@ -42,16 +48,21 @@ const postProcessData = (data, studyTotals) => {
     )[0] || []
 
   const notAvailableArray = largestHorizontalSection.map((studySection) => {
-    const totals = studyTotals[studySection.study]
-    const count = totals.targetTotal ? totals.targetTotal - totals.count : 0
+    const studySectionTotals = studyTotals[studySection.study]
+    const count = studySectionTotals.targetTotal
+      ? studySectionTotals.targetTotal - studySectionTotals.count
+      : 0
 
     return {
       color: 'grey',
       count,
       valueLabel: 'N/A',
       study: studySection.study,
       studyTarget: '',
-      percent: studyCountsToPercentage(count, totals.targetTotal),
+      percent: studyCountsToPercentage(
+        count,
+        studySectionTotals.targetTotal || studySectionTotals.count
+      ),
```

Comment:
do we want to use `count` if `targetTotal` is `0`, or only if it's not defined?
e.g. should this use nullish coalescing?
```suggestion
      percent: studyCountsToPercentage(
        count,
        studySectionTotals.targetTotal ?? studySectionTotals.count
      ),
```

Reply:
good point. The target should never be 0 but I guess if it is we'd want to us that.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/159/commits/e36b5e4e30a6e7c4f6cbdf00d8a74e611ffcd3fa